text/en/chapters/data-representation.md

```diff
@@ -682,6 +682,38 @@ The following interactive can be used to upload your own image, and experiment w
 
```

Comment:
This section is obviously going to merge conflict, can it please be put **after** the colour section, seeing as it is very advanced material, and I don't want it freaking out the NCEA students?

Another thought I have is, it will need to reference the content in the programming languages chapter, which gives an introduction to machine code. That chapter will help them understand what machine code is and how it relates to higher level programs, and then this content explains how it is represented using bits.